docs/editing_guidelines.md

```diff
@@ -90,6 +90,25 @@ If the main source for an assertion is a term in another ontology, the short ide
 
 ORCID identifiers may also be used when the only available source for an assertion is an individual researcher. This should be done sparingly.
 
+**Technical details of adding a cross-reference using Protégé**:
+
+For all: In the "Create Annotation" window, select the annotation property **database_cross_reference**.
+
+If the cross-reference can be represented as a CURIE and is not an ORCID: Enter the CURIE, using the bioregistry standard prefix, as a Value on the "Literal" tab. Leave Datatype empty.
+
+Non-CURIE values (e.g., URLs) are discouraged, but when used are also entered the same way, i.e., as a literal string with no Datatype selected.
```

Comment:
I would recommend to forbid them, and use rdfs:seeAlso for URLs. A URL is not a "database cross reference" even when the meaning is stretched.